### PR TITLE
CVSB-1970

### PR DESCRIPTION
**Pull request by mgherceanu**

### CODE REVIEW

File: src/pages/testing/vehicle/vehicle-history/vehicle-history.scss [hunk 1]
```diff
@@ -37,4 +37,17 @@ page-vehicle-history {
...
+  .no-result {
+    margin: auto;
+    width: 250px;
+    max-height: 50%;
+    text-align: center;
+    h3 {
+      color: $color-grey-1;
+      font-size: $font-size-medium;
+      font-weight: $font-weight-semi-bold;
+      font-family: "SF Pro Text",  Gadget, sans-serif;
```
Comment (danieliordache) [inline src/pages/testing/vehicle/vehicle-history/vehicle-history.scss]: use a variable

File: src/pages/testing/vehicle/vehicle-history/vehicle-history.html [hunk 2]
```diff
@@ -57,4 +57,8 @@
     </ng-container>
   </ion-list>
 
+  <div *ngIf="testResultHistory.length == 0" class="no-result">
```
Comment (danieliordache) [inline src/pages/testing/vehicle/vehicle-history/vehicle-history.html]: use js falsy values

File: src/pages/testing/vehicle/vehicle-history/vehicle-history.html [hunk 3]
```diff
@@ -13,7 +13,7 @@
       {{ vehicleData.vrm }}
     </div>
 
-  <ion-list class="tests-list">
+  <ion-list class="tests-list" *ngIf="testResultHistory.length > 0">
```
Comment (danieliordache) [inline src/pages/testing/vehicle/vehicle-history/vehicle-history.html]: use js thruthy values

File: src/pages/testing/vehicle/vehicle-history/vehicle-history.scss (hunk 1 above)
Comment (mgherceanu) [reply src/pages/testing/vehicle/vehicle-history/vehicle-history.scss]: DONE

File: src/pages/testing/vehicle/vehicle-history/vehicle-history.html (hunk 3 above)
Comment (mgherceanu) [reply src/pages/testing/vehicle/vehicle-history/vehicle-history.html]: done

File: src/pages/testing/vehicle/vehicle-history/vehicle-history.html (hunk 2 above)
Comment (mgherceanu) [reply src/pages/testing/vehicle/vehicle-history/vehicle-history.html]: done

**Pull request merged**